Dockerfile

```diff
@@ -9,6 +9,8 @@
 ARG VERSION
 FROM docker.io/mondoo/cnspec:${VERSION} AS root
 
+RUN ln -s /usr/local/bin/cnspec /usr/local/bin/mondoo
```

Comment:
what happens if mondoo is not present in /usr/local/bin? does it just keep going and doesn't care? im assuming so

Reply:
well, what we're doing here is just adding the symlink /usr/local/bin/mondoo -> /usr/local/bin/cnspec 

not changing the entrypoint of the container, so it will continue to default to launching `cnspec`

since we're building on top of the cnspec container image, i'm hoping it is safe to assume that i can make a symlink named /usr/local/bin/mondoo that just blindly points to where cnspec is supposed to be

this symlink is the case we discussed earlier today where we're not using the "big wrapper". this is just the smallest convenience for the off-chance that someone has been launching the mondoo/client container by specifying the path to the mondoo binary like so:

```
$ podman run --rm --entrypoint "mondoo" mondoo/client:custom status
→ no Mondoo configuration file provided. using defaults
→ Platform:     alpine
→ Version:      3.16.4
→ Hostname:     aae7b6e3bc41
→ IP:           10.0.2.100
→ Time:         2023-03-16T19:57:10Z
→ Version:      8.0.0 (API Version: 8)
→ API ConnectionConfig: https://us.api.mondoo.com
→ API Status:   SERVING
→ API Time:     2023-03-16T19:57:10Z
→ API Version:  8
→ Owner:
x client is not registered
x could not connect to mondoo platform
```